samples/boot/oauth2-integration/authorizationserver/src/main/java/sample/config/AuthorizationServerConfig.java

```diff
@@ -64,6 +67,15 @@ public CryptoKeySource keySource() {
 		return new StaticKeyGeneratingCryptoKeySource();
 	}
 
+	@Bean
+	public ProviderSettings providerSettings() {
```

Comment:
Please revert the changes as `ProviderSettings` is not required

Reply:
Should we keep it anyway ? Otherwise we wouldn't have the OpenID Provider Configuration endpoint available (and later OAuth 2.0 Server Metadata endpoint).

We could add a comment explaining _why_ we have this Bean.

Reply:
Let's remove it since we don't have OIDC auth implemented yet #53. I'll be starting #53 next week and I'll be sure to add this back to the sample as part of that PR.